go.mod

```diff
@@ -18,15 +20,15 @@ require (
 	github.com/onsi/ginkgo v1.8.0
 	github.com/onsi/gomega v1.5.0
 	github.com/pkg/errors v0.8.1
-	github.com/prometheus/client_golang v0.9.3-0.20190127221311-3c4408c8b829
+	github.com/prometheus/client_golang v0.9.4
```

Comment:
We want `github.com/prometheus/client_golang v0.9.3-0.20190127221311-3c4408c8b829`. This uses protobuf 1.2.0. Don't change it.